editor/editor_node.cpp

```diff
@@ -3516,6 +3516,10 @@ void EditorNode::_remove_scene(int index, bool p_change_tab) {
 	// FIXME: Ideally the cache should never be cleared and only updated on per-script basis, when an icon changes.
 	editor_data.clear_script_icon_cache();
 
+	// Mark the scene tab index as removing, so this scene will get ignored when calling EditorSceneTabs::update_scene_tabs()
```

Comment:
```suggestion
	// Mark the scene tab index as removing, so this scene will get ignored when calling EditorSceneTabs::update_scene_tabs().
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide).